src/main/java/cubicchunks/asm/mixin/core/client/MixinViewFrustum_BiggerRenderChunk.java

```diff
@@ -103,18 +124,15 @@ private void updateChunkPositionsInject(double viewEntityX, double viewEntityZ,
                 }
             }
         }
-        cbi.cancel();
     }
 
     @Inject(method = "getRenderChunk", at = @At(value = "HEAD"), cancellable = true, require = 1)
     private void getRenderChunkInject(BlockPos pos, CallbackInfoReturnable<RenderChunk> cbi) {
-        if (!((ICubicWorld) world).isCubicWorld()) {
-            return;
```

Comment:
This stays. You don't change rendering in vanilla worlds. 32x32x32 cubes can be an exception for vanilla but it should still do it the vanilla way, which not moving the rendered area up/down with the player.

Reply:
I do. Bigger render chunks works fine with vanilla chunks as well. Why limit that feature to only cubic worlds? This brings a lot of `.isCubicWorld()` to all constant modifiers.

Reply:
Bigger chunks should work in vanilla, but without making any part of the rendering use different RenderChunks depending on player Y.

Reply:
A-a-a... got it.